### PR TITLE
Add sim threshold table to complete user story #12

### LINKED ISSUE #12: Display a list of individuals that have similarity with an other above a selected threshold. This way the user can obtain a list of all users with plagiarized submissions quickly and easily. Justification: In the case of professors/teachers, this feature would allow them to see what students had the highest plagiarism scores and provide grades accordingly without sifting through scores below the threshold. This feature is quite impactful as it can lend itself to the ease of use of the project. Given the time frame this can be a feature that we can begin implementation on if time permits.

### PR DESCRIPTION
**Pull request by JavaNoTea**
# Pull request to resolve user story #12: 
### What?!

This adds a second search bar under the original search bar. The new bar takes in an integer input. This integer is the similarity threshold, all submissions that have a simliarity of equal or lower values will be filtered out of the list.

---
### Why?!

This allows for quick filtering of submissions. For example, if you are a professor and you want to fail anyone who has more than 50% similiarity, you just enter 50 and fail everyone in the list. Failing has never been so easy!

---
### How?!

The comparsion table component now has a brand new component in it titled SimThresBarComponent.vue. SimThresBarComponent.vue is a search bar input that changes the threshold number that is constantly be monitored by ComparisonTableFilter.vue who is ready to update the table immediately on any change.

---
#### Files modified:
- report-viewer/src/components/ComparisonTableFilter.vue
- report-viewer/src/components/ComparisonsTable.vue
- report-viewer/src/components/SimThresBarComponent.vue

#### This pull request resolves #12 user story by:

- **Resolve issue #9:** Add threshold input and populate list button to the bottom of the threshold list window
- **Resolve issue #10:** Implementing threshold list functionality/filtering by threshold value
- **Resolve issue #11:** (Originally) Add a list window to hold the users above a selected similarity threshold
    (Modified to) Combine filters into one list window
---
### TL;DR:
All together this pull request implements the #12 user story. That is it adds a second search bar that takes in an integer and removes any submission from the list that have a similarity score equal or less than that integer. This allows for two search criteria to filter the list simultaneously.

### CODE REVIEW

**Review by Jerry4753 — APPROVED** [review]
good job creating a separate component for the threshold bar. Everything else also looks good to me.

**Review by asturnley — APPROVED** [review]
Looks good! Great job adding in the threshold set by user when comparing submissions.
